tests/cross_functional/system_test/test_mcg_bucket_notification.py

```diff
@@ -0,0 +1,305 @@
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import (
+    ignore_leftover_label,
+    polarion_id,
+    mcg,
+    magenta_squad,
+    system_test,
+)
+from ocs_ci.ocs.bucket_utils import (
+    write_random_test_objects_to_bucket,
+    s3_delete_object,
+    s3_put_object_tagging,
+    expire_objects_in_bucket,
+)
+from ocs_ci.ocs.resources.bucket_notifications_manager import (
+    BucketNotificationsManager,
+    logger,
+)
+from ocs_ci.ocs.resources.mcg_lifecycle_policies import (
+    LifecyclePolicy,
+    ExpirationRule,
+    LifecycleFilter,
+)
+from ocs_ci.ocs.resources.pod import (
+    get_pod_node,
+    get_pods_having_label,
+    Pod,
+    get_noobaa_core_pod,
+    get_noobaa_db_pod,
+    get_noobaa_endpoint_pods,
+)
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+@mcg
+@system_test
+@magenta_squad
+@ignore_leftover_label(constants.CUSTOM_MCG_LABEL)
```

Comment:
You will need to add a few skip decorators here:
* `skipif_disconnected_cluster` and `skipif_proxy_cluster` because the AMQ setup clones a repo from github
* `skipif_noobaa_external_pgsql` and `skipif_external_mode` because of the PGSQL queries for manual expiration

It might be possible to find smarter solutions than to skip, but that would be outside of this PR's scope

Reply:
Done